frontend/src/app/shared/form-control-validation.directive.ts

```diff
@@ -10,7 +10,8 @@ import { ValdemortConfig } from 'ngx-valdemort';
  * and the error message appear together).
  */
 @Directive({
-  selector: '.form-control,[ngbRadioGroup],.custom-select'
+  selector: '.form-control,[ngbRadioGroup],.custom-select',
```

Comment:
This is an error (that I did during the bootstrap 5 migration): there is no .custom-select anymore (it's .form-select IIRC). And there is no ngbRadioGroup anymore either IIRC (but maybe we don't use bootstrap radio button groups, so we can drop this selector)